src/components/Header.spec.js

```diff
@@ -0,0 +1,25 @@
+import {render, screen} from '@testing-library/react';
+//import userEvent from '@testing-library/user-event';
+
+import Header from './Header.js';
+
+describe('InputDataDialog', () => {
+
+    it('is an accessible form', () => {
+        render(<Header />);
+        screen.getByLabelText('Please insert data - only numbers allowed:');
+        screen.getByRole('button', {name: 'Submit'});
+        screen.getByRole('spinbutton');
+      });
+
+    /*
+    it('testing submit', async () => {
+        const user = userEvent.setup();
+        const callback = jest.fn();
+        render(<InputDataDialog updateInput={callback} />);
+    
+        const input = screen.getByLabelText('Please insert data - only numbers allowed:');
+        await user.type(input, '7{Enter}');
+        expect(callback).toHaveBeenCalledWith('7');
+      });*/
```

Comment:
Please remove unused code before merging!
```suggestion
import {render, screen} from '@testing-library/react';

import Header from './Header.js';

describe('InputDataDialog', () => {

    it('is an accessible form', () => {
        render(<Header />);
        screen.getByLabelText('Please insert data - only numbers allowed:');
        screen.getByRole('button', {name: 'Submit'});
        screen.getByRole('spinbutton');
      });
```